network/notifee.go

```diff
@@ -26,7 +26,12 @@ type NotifeeService struct {
 func newNotifeeService(ctx context.Context, host lp2phost.Host, eventChannel chan<- Event, peerMgr *peerMgr,
 	protocolID lp2pcore.ProtocolID, bootstrapper bool, log *logger.SubLogger,
 ) *NotifeeService {
-	eventSub, err := host.EventBus().Subscribe(lp2pevent.WildcardSubscription)
+	evts := []interface{}{
```

Comment:
is that `evt` correct? or should be `event`?

Reply:
Done